app/assets/stylesheets/application.scss

```diff
@@ -12,4 +12,16 @@
  *
  */
 
-@import "bootstrap";
\ No newline at end of file
+@import "bootstrap";
+
+.btn-purple {
```

Comment:
I would not set the class name to attached to a css property value, what would happen if tomorrow you decide purple is not the right color and change to, let's say orange, you will have to update the class name in order for the button to make sense at all....